src/elements/Header/_Header.js

```diff
@@ -1,42 +1,98 @@
 import cx from 'classnames'
-import React, { createElement, Component, PropTypes } from 'react'
-
+import React, { PropTypes } from 'react'
 import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+  iconPropRenderer,
+  imagePropRenderer,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+function _Header(props) {
+  const {
+    _sdClass, _headerElement,
+    left, center, right, justified, dividing, block, attached, floating,
+    icon, image, children, className,
+  } = props
+
+  const classes = cx(
+    _sdClass, 'ui',
+    useKeyOnly(left, 'left aligned'),
+    useKeyOnly(center, 'center aligned'),
+    useKeyOnly(right, 'right aligned'),
+    useKeyOnly(justified, 'justified'),
+    useKeyOnly(dividing, 'dividing'),
+    useKeyOnly(block, 'block'),
+    useKeyOnly(attached, 'attached'),
```

Comment:
The [attached](http://semantic-ui.com/elements/header#attached) option appears as `top attached`, `attached`, and `bottom attached` html classes.  In this case the key _or_ the value and key are used.  They are express in the API as:

```js
<Header attached />
<Header attached='top' />
<Header attached='bottom' />
```

Then in the className buildup as:

```js
cx(
  useKeyOrValueAndKey(attached, 'attached'),
)
```

This method will use the boolean value (attached) as the className if no value was provided.  It will also handle the value if provided.  Hence the awkward but accurate naming "use key _or_ value and key".

***

For comparison, the current implementation would always produce this for all the above API permutations:

```html
<div class="ui attached header"></div>
```

Because only the key "attached" is being used.  Use `useKeyOnly` for boolean style classes, classes that can appear in tandem with other classes, are either present or not, and do not have any combination classes.